tests/bud.bats

```diff
@@ -3719,3 +3719,155 @@ _EOF
   target=bud-group
   run_buildah build --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/group
 }
+
+@test "bud-with-mount-bind-from-like-buildkit" {
+  skip_if_no_runtime
+  skip_if_in_container
+  mkdir ${TESTDIR}/bud
+  cp -R ${TESTSDIR}/bud/buildkit-mount-from ${TESTDIR}/bud/buildkit-mount-from
+  # build base image which we will use as our `from`
+  run_buildah build -t buildkitbase --signature-policy ${TESTSDIR}/policy.json -f ${TESTDIR}/bud/buildkit-mount-from/Dockerfilebuildkitbase ${TESTDIR}/bud/buildkit-mount-from/
+  # try reading something from persistent cache in a different build
+  run_buildah build -t testbud --signature-policy ${TESTSDIR}/policy.json -f ${TESTDIR}/bud/buildkit-mount-from/Dockerfilebindfrom
+  expect_output --substring "hello"
+  run_buildah rmi -f buildkitbase
+  run_buildah rmi -f testbud
+}
+
+@test "bud-with-mount-bind-from-without-source-like-buildkit" {
+  skip_if_no_runtime
+  skip_if_in_container
+  mkdir ${TESTDIR}/bud
+  cp -R ${TESTSDIR}/bud/buildkit-mount-from ${TESTDIR}/bud/buildkit-mount-from
+  # build base image which we will use as our `from`
+  run_buildah build -t buildkitbase --signature-policy ${TESTSDIR}/policy.json -f ${TESTDIR}/bud/buildkit-mount-from/Dockerfilebuildkitbase ${TESTDIR}/bud/buildkit-mount-from/
+  # try reading something from persistent cache in a different build
```

Comment:
Suggest "another image" instead of "persistent cache" here.

Reply:
fixed